blocks/explore-cards/explore-cards.css

```diff
@@ -1 +1,194 @@
+    flex-direction: row;
+    align-items: center;
+}
+
+.explore-cards {
+    width: 100%;
+    padding: var(--spacing-48) 0 var(--spacing-64) 0;
+}
+
+.heading {
+    margin-bottom: var(--spacing-20);
+}
+
+.heading p {
+    font-size: var(--heading-font-size-m);
+    line-height: 30.8px;
+}
+
+.explore-card-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: var(--spacing-20);
+    width: calc(100vw - 68px);
+    overflow-x: scroll;
+}
+
+.explore-card {
+    position: relative;
+    border-radius: 10px;
+    height: 400px;
+    min-width: 300px;
+}
+
+.card-icon.icon-video {
+    width: 72px;
+    height: 72px;
+    background-color: var(--background-color);
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 20px 0;
+    background-image: url("/icons/playbutton.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+}
+
+.card-icon.icon-audio {
+    width: 72px;
+    height: 72px;
+    background-color: var(--background-color);
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 20px 0;
+    background-image: url("/icons/podcast.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+}
+
+.explore-card a {
+    text-decoration: none;
+    color: var(--color-white);
+    display: block;
+    height: 100%;
+
+    div {
+        height: 100%;
+    }
+}
+
+.explore-card picture img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    display: block;
+}
+
+.card-content {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    padding: var(--spacing-20);
+    background: linear-gradient(180deg, rgb(1 39 64 / 0%) 39.98%, #012740 78.99%);
+    color: var(--color-white);
+    z-index: 1;
+}
+
+.card-title p {
+    font-size: var(--heading-font-size-xs);
+    font-weight: 700;
+    line-height: 24px;
+}
+
+.story-type {
+    font-size: var(--body-font-size-xs);
+    font-weight: 700;
+    line-height: 13.2px;
+    letter-spacing: 0.06em;
+}
+
+.story-type p {
+    font-size: var(--body-font-size-xs);
+    font-weight: 700;
+    line-height: 13.2px;
+    letter-spacing: 0.06em;
+}
+
+.card-content .card-title h3 {
+    font-size: var(--heading-font-size-xxs);
+    font-weight: 700;
+    line-height: 24px;
+}
+
+@media (width >=768px) {
+    .explore-cards {
+        padding: var(--spacing-64) 0 var(--spacing-80) 0;
+    }
+
+    .heading p {
+        font-size: var(--heading-font-size-l);
+        line-height: 35.2px;
+    }
+
+    .explore-card-container {
+        display: grid;
+        grid-template-columns: 1fr;
+        gap: var(--spacing-20);
+        width: unset;
+        overflow-x: unset;
+    }
+
+    .explore-card {
+        position: relative;
+        border-radius: 10px;
+        height: 350px;
+        min-width: unset;
+    }
+
+    .card-title p {
+        font-size: var(--heading-font-size-xxs);
```

Comment:
some properties are already defined at global level and do not need to mention again, like
```
font-weight: 700;
line-height: 24px;
```